mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -138,6 +147,7 @@ trait Completions { this: MetalsGlobal =>
     if (sym.isSynthetic) relevance |= IsSynthetic
     if (sym.isDeprecated) relevance |= IsDeprecated
     if (isEvilMethod(sym.name)) relevance |= IsEvilMethod
+    if (isNamedArgument) relevance |= IsNotLocalByBlock
```

Comment:
This might need maybe a lower score than IsNotLocalByBlock - maybe just a new lowest score?